cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CaseExpressionsTest.scala

```diff
@@ -39,8 +39,11 @@ class CaseExpressionsTest extends DocumentingTest {
                   #  (bob)-[:KNOWS]->(daniel),
                   #  (charlie)-[:KNOWS]->(daniel),
                   #  (bob)-[:MARRIED]->(eskil)""".stripMargin('#'))
-    p("""Generic conditional expressions may be expressed using the well-known `CASE` construct.
+   p("""Generic conditional expressions may be expressed using the `CASE` construct.
```

Comment:
I think the previous indentation worked better.
```suggestion
    p("""Generic conditional expressions may be expressed using the `CASE` construct.
```